gpib_server.py

```diff
@@ -165,6 +165,11 @@ def write(self, c, data):
         """Write a string to the GPIB bus."""
         self.getDevice(c).write(data)
 
+    @setting(8, data='s', returns='')
```

Comment:
In the recent (scala) versions of the manager there is a new type `'y'` for binary data. The intent is that `'s'` is for text strings in utf8 encoding, while `'y'` can contain arbitrary bytes. Note that the `read_raw` setting below returns `'y'`; we should use that type here as well.